### ISSUE TITLE
support jac

### ISSUE THREAD

**Issue by qthedoc** [qthedoc/ez-optimize#3]
look into JAX for kw gradients

https://docs.jax.dev/en/latest/installation.html
https://github.com/jax-ml/jax/discussions/17820
